app/controllers/application_controller.rb

```diff
@@ -118,4 +118,19 @@ def set_locale
       I18n.locale = params[:locale] || I18n.default_locale
     end
 
+  def spam_revision
```

Comment:
Thanks for this followup fix -- I think we need a more descriptive method name, though -- you should be able to see in the notes_controller and questions_controller that there may be a redirect, and that there may be alerts. It's long, but can you rename it "alert_and_redirect_moderated"?